<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResearcherFormClass</class>
 <widget class="QMainWindow" name="ResearcherFormClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>981</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ResearcherForm</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>200</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTableView" name="tableViewIdeas">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>351</width>
      <height>411</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>184</width>
      <height>157</height>
     </rect>
    </property>
    <property name="title">
     <string>Add idea</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Title:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLineEdit" name="lineEditTitle"/>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Description:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="lineEditDesc"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Duration:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="lineEditDuration"/>
     </item>
     <item row="3" column="0" colspan="3">
      <widget class="QPushButton" name="pushButtonAdd">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButtonAccept">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>430</y>
      <width>161</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Accept</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDevelop">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>430</y>
      <width>171</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Develop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSaveAll">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>430</y>
      <width>331</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Save All</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableViewDevelop">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>10</y>
      <width>331</width>
      <height>411</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ResearcherForm.qrc"/>
 </resources>
 <connections/>
</ui>
